ovos_utils/process_utils.py

```diff
@@ -298,7 +298,7 @@ def init(cls):
     # Class constants
     DIRECTORY = None
     FILE = '/{}.pid'
-    LOG.info(f"Create PIDLock in: {DIRECTORY}")
+    LOG.debug(f"Create PIDLock in: {DIRECTORY}")
```

Comment:
this log should be removed, its at class level and its just spam, can also be moved to init instead